backend/dtos/common.py

```diff
@@ -1,51 +1,52 @@
 from typing import Any
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel as Model
+from pydantic import Field
 
 from backend.utils import snake_to_camel
 
 
-class AuthDTO(BaseModel):
-    access_token: str = Field(example="")
-    refresh_token: str = Field(example="")
-
-
-class BaseDTO(BaseModel):
+class BaseModel(Model):
```

Comment:
We have string from pydantic import BaseModel as Model. In one case we inherit from Mode in other cases from BaseModel. Why?

Reply:
we redefine pydantic class BaseModel, so that all the DTOs use the redefined class with extra params. Because we need this params in each DTO, it's better to define a base class they all inherit from instead of passing those params in each DTO